feature-libs/user/profile/components/register/register.component.ts

```diff
@@ -65,6 +65,7 @@ export class RegisterComponent implements OnInit, OnDestroy {
         this.registerComponentService.generateAdditionalConsentsFormControl?.() ??
         this.fb.array([]),
       termsandconditions: [false, Validators.requiredTrue],
+      captcha: [false, Validators.requiredTrue],
```

Comment:
This is a breaking change. No user will be able to register anymore without enabling captcha. So we need to make this validator opt-in (eg. with feature flags or checking whether some captcha service has been included) until the next major version `7.0`.

Reply:
Thx for your info. I will do it after confirming the release version with PO.